### ISSUE TITLE
[MU4 Task] Request: 'Break Multi-measure rests' should apply to all parts

### ISSUE THREAD

**Issue by eitanmuir** [musescore/MuseScore#15753]
Hey MU4 team,

I'm working on a moderately large project and have encountered a functional UX issue. When 'Break Multi-measure rest' is checked in Measure Properties in the score, it is not automatically applied to parts. (As compared with Sibelius, where 'break multi-measure rests' in the score applies to all parts unless the user decides otherwise.)

Manually breaking multi-measure rests in each part as required opens the possibility for 1) user error, and 2) format issues arising within already-engraved parts. For large-scale works this can become quite annoying.

Hoping it's possible to sync multi-measure rests in parts for 4.x?

**Comment by MarcSabatella** [comment]
I agree this probably makes sense to do by default, although an issue is, if cases arise where it *doesn't* make sense, it would not be possible to override this.  that is one reason MuseScore has historically been conservative about which properties are linked.

In theory a plugin could copy this property to all parts.  But meanwhile, is there a reason you are trying to break rests at a place where there isn't a double bar, rehearsal mark, tempo change, or other marking that does this automatically?  It should be very rare to need to set this property explicitly.  And in any case, it suggests a smile possible workaround - add an invisible rehearsal marking.  You'll need to make it invisible also individually in the parts but at least that's easier.

**Comment by eitanmuir** [comment]
> I agree this probably makes sense to do by default, although an issue is, if cases arise where it _doesn't_ make sense, it would not be possible to override this. that is one reason MuseScore has historically been conservative about which properties are linked.

Could Link Multi-measure Breaks be a Style setting, perhaps?

> is there a reason you are trying to break rests at a place where there isn't a double bar, rehearsal mark, tempo change, or other marking that does this automatically? It should be very rare to need to set this property explicitly. 

Writing for amateur gamelan octet (no conductor) with long syncopated phrases that start on the downbeat and end across barlines. Very few rehearsals before premiere - it helps to have the supporting instruments' rests broken up into phrases - easier for amateur players to learn the music in absence of a proper reference recording.

I find these kinds of breaks are really helpful for musical theatre, contemporary music, anything upbeat and syncopated really - where the rest isn't long enough to justify a written cue, but too long to be easy to sightread.

> add an invisible rehearsal marking. You'll need to make it invisible also individually in the parts but at least that's easier.

I guess that works for now!

**Comment by MarcSabatella** [comment]
It's possible that style settings would be a way to control this, but we really need a general solution not just for this property but for many others.

In musical theater etc, you should definitely be using double bars and/or rehearsal to mark the important points in the piece, and that's it's where multimeasure rests should break.  There would be almost no reason in that context to ever break a multimeasure rest in a place that doesn't also deserve double bars andor rehearsal marks.
